content/blog/2020-05-04-covid-19-open-data/index.md

```diff
@@ -0,0 +1,95 @@
+---
+slug: "covid-19-open-data"
+title: Levaraging open data for research
+author:
+  - Maëlle Salmon
+  - Stefanie Butland
+  - Karthik Ram
+  - Noam Ross
+  - Mark Padgham
+  - Jeroen Ooms
+  - Carl Boettiger
+  - Scott Chamberlain
+date: 2020-05-04
+tags:
+  - packages
+  - R
+  - community
+  - reproducibility
+  - data
+  - data-access
+  - open-data
+description: "Covid-19 and Open Data from rOpenSci."
+# twitterImg: blog/2019/06/04/post-template/name-of-image.png
+---
+
+
+Covid-19 has changed all of our lives. For researchers in particular, field and lab based research may not be possible, derailing lives and long-term research projects. However, if you still have internet service, you can conduct research with the massive amount of open data available.
+
+What it takes to do research using open data probably will not require new skills beyond what most researchers already know. Instead of using primary research data collected in the field or lab however, researchers can instead use open data as their starting point.
+
+some other text ...?
+
+rOpenSci packages make open data easy to incorporate into your reproducibile R workflow. What R packages do we have and how much and what data do they provide? We'll answer these questions below.
+
+## A lot of data
+
+<!-- Below we'll cover a variety of research areas (see _Thematic Areas_),  -->
+
+rOpenSci packages give R users access to a whole lot of data. For illustrative purposes, here are a few of the packages we have covering a variety of different thematic areas:
+
+Biology
+
+* [rentrez][]/[phylotaR][]/[biomartr][]: very brief description of data size
+* [rgbif][]: very brief description of data size
+* [rebird][]/[auk][]: very brief description of data size
+* [taxize][]: very brief description of data size
+
+Social science
+
+* [essurvey][]: very brief description of data size
+* [comtradr][]: very brief description of data size
+
+Climate/Spatial
+
+* [MODIStsp][]: very brief description of data size
+* [rnoaa][]: very brief description of data size
+* [webchem][]: very brief description of data size
+
+Economics
+
+* [skynet][]: very brief description of data size
+
+
+## Thematic Areas
+
+We've invited some rOpenSci community members to summarize packages for accessing open data in each of a number of thematic areas. The thematic areas are: water, Antarctica, ecology, biodiversity, taxonomy, space and weather, and text-mining.
+
+_for each thematic area:_
+
+### Water
+
+**Author**: Jane Doe
+
+some text ... (max 1000 characters?)
+
+## Feedback
+
+Questions? thought? data you want that you can't access in R easily?
+
+
+[biomartr]: https://github.com/ropensci/biomartr
```

Comment:
We should link to docs websites 🙂 docs.ropensci.org/biomatr etc

Reply:
thanks